src/DependencyInjection/ConsoleErrorsExtension.php

```diff
@@ -40,12 +40,12 @@ public function loadInternal(array $mergedConfig, ContainerBuilder $container):
 		);
```

Comment:
Please YML->YAML should also be changed in examples in readme.md.

Reply:
And also please also all of the other relevant YAML DI changes.

Reply:
Fixed.